wiki/Tournaments/SS_Tournament/2018/en.md

```diff
@@ -0,0 +1,286 @@
+| MFWPTG                    | ![][flag_US] [Corim](https://osu.ppy.sh/users/4242767), ![][flag_US] [Exarch](https://osu.ppy.sh/users/5370153), ![][flag_US] [Noname0](https://osu.ppy.sh/users/280147), ![][flag_CA] [ChronoNig](https://osu.ppy.sh/users/3545323) |
+| discord.gg/hhVMaZ4        | ![][flag_TH] [Trigonoculus](https://osu.ppy.sh/users/7627013), ![][flag_SG] [moosepi](https://osu.ppy.sh/users/1868745), ![][flag_ID] [Landegre-](https://osu.ppy.sh/users/3654630), ![][flag_SG] [Jolene](https://osu.ppy.sh/users/2672025) |
+| Die Augen                 | ![][flag_DE] [Refrum](https://osu.ppy.sh/users/7064509), ![][flag_DE] [Auge](https://osu.ppy.sh/users/7115794), ![][flag_DE] [Xenorin](https://osu.ppy.sh/users/6759462), ![][flag_DE] [tagorin](https://osu.ppy.sh/users/7243516) |
+| Pszne Rzybki              | ![][flag_PL][MajsterDeeJay](https://osu.ppy.sh/users/6329597), ![][flag_PL] [-Mario-](https://osu.ppy.sh/users/4950514), ![][flag_PL] [Hatiso](https://osu.ppy.sh/users/9011751), ![][flag_PL] [LowAccDeadMan](https://osu.ppy.sh/users/8417559) |
+| Jorge                     | ![][flag_BR] [- Emerald](https://osu.ppy.sh/users/6488891), ![][flag_BR] [Crimson Dragon](https://osu.ppy.sh/users/5085555), ![][flag_BR] [Astris](https://osu.ppy.sh/users/6173856) |
+| Adrenaline Rush           | ![][flag_CA] [Xoul] (https://osu.ppy.sh/users/7856835), ![][flag_US] [[ Scarlet Red ]](https://osu.ppy.sh/users/3257340), ![][flag_US] [Arsene](https://osu.ppy.sh/users/4427983), ![][flag_US] [Tobirama Senju](https://osu.ppy.sh/users/3188127) |
+| gommehd.net               | ![][flag_DE] [[Swegloerd]](https://osu.ppy.sh/users/7382962), ![][flag_DE] [Amanion](https://osu.ppy.sh/users/7382973), ![][flag_DE] [prhtnsm](https://osu.ppy.sh/users/6702666), ![][flag_DE] [Sandwich](https://osu.ppy.sh/users/7528409) |
+| jenny tools               | ![][flag_RO] [Chamosiala](https://osu.ppy.sh/users/1469892), ![][flag_LT] [Auji](https://osu.ppy.sh/users/4114438), ![][flag_ES] [yotampoco](https://osu.ppy.sh/users/7073613), ![][flag_IT] [Koba](https://osu.ppy.sh/users/4448118) |
+| interatlantic baguettes   | ![][flag_CA] [DivineRose](https://osu.ppy.sh/users/8151359), ![][flag_FR] [Hichinoro](https://osu.ppy.sh/users/5410051), ![][flag_GB] [Cornfield](https://osu.ppy.sh/users/4087422) |
+| Tsutaerarenakatta         | ![][flag_PL] [Reifa](https://osu.ppy.sh/users/5725033), ![][flag_PL] [-Sylvari](https://osu.ppy.sh/users/3493804), ![][flag_PL] [Cypriane](https://osu.ppy.sh/users/7703578), ![][flag_PL] [ChainedTomb](https://osu.ppy.sh/users/4966523) |
+| Squeaky clean             | ![][flag_MX] [Janssen](https://osu.ppy.sh/users/2279351), ![][flag_US] [mniminwoo], ![][flag_MX] [-Hebel-](https://osu.ppy.sh/users/6169483), ![][flag_US] [Mlaw22](https://osu.ppy.sh/users/3126596) |
```

Comment:
`mniminwoo` is missing a link

Reply:
fixed all above issues, and dont appologise its much appreciated! sorry for having all the mistakes you have to spot in the first place ;p